metadata-ingestion-examples/mce-cli/build.gradle

```diff
@@ -0,0 +1,40 @@
+plugins {
+  id 'org.springframework.boot'
+  id 'java'
+  id 'pegasus'
+}
+
+dependencies {
+  compile project(':metadata-utils')
+  compile project(':metadata-builders')
+  compile project(':metadata-dao-impl:kafka-producer')
+  compile project(':metadata-events:mxe-schemas')
+  compile project(':metadata-ingestion-examples:common')
+
+  dataModel project(':metadata-models')
+
+  compile spec.product.pegasus.restliServer
+  compile externalDependency.javaxInject
+  compile externalDependency.kafkaAvroSerde
+  compile externalDependency.lombok
+  compile externalDependency.picocli
+  compile externalDependency.springBeans
+  compile externalDependency.springBootAutoconfigure
+  compile externalDependency.springCore
+  compile externalDependency.springKafka
+
+  runtime externalDependency.logbackClassic
+
+  annotationProcessor externalDependency.lombok
+  annotationProcessor externalDependency.picocli
+}
+
+bootRun {
+  if (project.hasProperty('args')) {
+    args project.args.split(',')
```

Comment:
Seems a bit hacky but no idea how to improve it :P

Reply:
Try this? https://stackoverflow.com/a/51407975

Reply:
Ah, yeah, seems supported by spring

https://docs.spring.io/spring-boot/docs/current/gradle-plugin/reference/html/#running-your-application-passing-arguments